common/src/jni/main/cpp/org_conscrypt_NativeCrypto.cpp

```diff
@@ -29,7 +29,12 @@
 
 #define LOG_TAG "NativeCrypto"
 
-#include <arpa/inet.h>
+#ifdef _WIN32
+    #include <Winsock2.h>
```

Comment:
This should be lowercase in case we compile against MinGW which is usually compiled with case-sensitive filesystems.